build.gradle

```diff
@@ -54,6 +54,7 @@ allprojects {
                 check("PreferSafeLoggableExceptions", CheckSeverity.OFF)
                 check("PreferSafeLoggingPreconditions", CheckSeverity.OFF)
                 check("PreconditionsConstantMessage", CheckSeverity.OFF)
+                check("GradleCacheableTaskAction", CheckSeverity.OFF)
```

Comment:
We can re-enable this as soon as this repo upgrades to a version of baseline that includes this PR.

Reply:
It's really scary that a bump in `error_prone_annotations` alone (which looks like it should be a sort of API, not implementation) changes the implementation logic.